orga/tests/integration/components/sco-organization-participant/list_test.js

```diff
@@ -370,4 +370,31 @@ module('Integration | Component | ScoOrganizationParticipant::List', function (h
         .exists();
     });
   });
+
+  module('when there is no participants', function ({ beforeEach }) {
+    beforeEach(function () {
+      const store = this.owner.lookup('service:store');
+      this.set('students', [
+        store.createRecord('sco-organization-participant', {
+          lastName: 'La Terreur',
+          firstName: 'Gigi',
+          birthdate: '2010-01-01',
+          email: null,
+          username: null,
+          isAuthenticatedFromGar: true,
+        }),
+      ]);
+    });
+
+    test('it should display a message telling so there is no participants', async function (assert) {
```

Comment:
**Suggestion :** _'it should display a message telling so there is no participants'_

Reply:
je suppose que c'est => it should display a message telling if there is no participants
done